### PR TITLE
chore: update README with latest information

### PR DESCRIPTION
**Pull request by connorsmith256**
## Feature or Problem
This updates the README in a few ways:
- distinguish between the stable and experimental ABIs
- mark OTP host as deprecated and un-marks the Rust host as experimental
- documents some more crates we've moved into this repo, and a couple others that aren't in this repo, for increased discoverability
- adds some greater visual separation between sections

For a preview of the changes, see https://github.com/connorsmith256/wasmCloud/tree/chore/update-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,54 +76,72 @@ We host weekly community meetings at 1pm EST on Wednesdays. These community meet
...
+### 🦀 Rust Runtime
 
-Rust runtime is under heavy development at the root of this repository.
+wasmCloud's [standard runtime](./crates/runtime) is built in Rust for its zero-cost abstractions, safety, security, and WebAssembly support.
 
 ### 🕸 JavaScript Runtime (`Experimental`)
 
 For running a wasmCloud host in a browser or embedding in a JavaScript V8 host, use the [JavaScript Runtime](https://github.com/wasmCloud/wasmcloud-js)
 
+~~### ☁️ Elixir/OTP Runtime (`Deprecated`)~~
```
Comment (thomastaylor312) [inline README.md:109]: Not sure I really like the strikethrough on this section, it also causes the header to not render

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
One minor formatting issue

File: README.md (hunk 1 above)
Comment (connorsmith256) [reply README.md:109]: Oh good catch, didn't notice that

File: README.md [hunk 2]
```diff
@@ -72,54 +76,74 @@ We host weekly community meetings at 1pm EST on Wednesdays. These community meet
...
 
-`wasmcloud_actor` is a wasmCloud actor library written in Rust and available at [`./crates/actor`](./crates/actor).
+### `wasmcloud_actor` (`Experimental`)
 
-It provides functionality, which facilitates building of wasmCloud actors.
+[`wasmcloud_actor`](./crates/actor) is a wasmCloud actor library written in Rust which facilitates building of wasmCloud actors.
 
 The API of the crate matches closely what [`wit-bindgen`](https://github.com/bytecodealliance/wit-bindgen) would generate, meaning that one can switch from using plain `wit-bindgen`-generated bindings to `wasmcloud_actor` (and back) with minimal or no code changes.
 
 Latest documentation is available at [wasmcloud.github.io/wasmCloud/wasmcloud_actor](https://wasmcloud.github.io/wasmCloud/wasmcloud_actor/).
```
Comment (rvolosatovs) [inline README.md:141]: This link is actually broken, should we just remove it? (for now)

**Review by thomastaylor312 — DISMISSED** [review]
Marking as good to go so as to not block, but please don't merge before addressing @rvolosatovs' comment

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**